### PR TITLE
Budget manager: added docstring to the methods

### PR DESCRIPTION
**Pull request by Akshayan03**

### CODE REVIEW

**Review by vandanppatel — APPROVED** [review]
Great adding docstrings, makes it easier for others to see what your program does as well as makes it easier for you to also to go back and make any edits if necessary

File: src/main/java/Entities/Budget.java [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 public class Budget {
     protected double initialBudget;
-    protected static double currentBudget;
+    protected double currentBudget;
 
     public Budget () {
     }
 
-    public Budget (double initialBudget, double currentBudget){
+    public Budget (double initialBudget){
```
Comment (Aviral-03) [inline src/main/java/Entities/Budget.java:10]: Darpan changed similar thing in his pull request this will create conflicts

**Review by darpanmishra — APPROVED** [review]
Looks good to me, now we can focus on implementing clean architecture throughout our use cases by utliizing presenters and controllers.

**Review by Aviral-03 — APPROVED** [review]
Looks like all the test passed for Autograding. Good job on adding doc strings to each document. Kindly write tests and starting refactoring your code according to clean architecture.

**Pull request merged**